app/views/unattended/provisioning_templates/host_init_config/host_init_config_default.erb

```diff
@@ -9,11 +9,31 @@ oses:
 - Ubuntu
 -%>
 #!/bin/bash
-
 set -e
 
-# This script enrolls a host with Foreman.
-# Host: <%= @host.name %>
+<% built_https = foreman_url('built').start_with?('https') -%>
```

Comment:
In other places we have a convention to set variables before the actual script:
```suggestion
-%>
<% built_https = foreman_url('built').start_with?('https') -%>
#!/bin/bash

set -e
```

(it doesn't render well at all, but the idea is to place it between `<%# -%>` and the shebang)